src/lib/types.ts

```diff
@@ -1,29 +1,36 @@
 export interface ArtistProps {
-    id: number,
-    name: string,
-    description: string,
-    url: string,
-    image: string,
+  id: number;
+  name: string;
+  description: string;
+  url: string;
+  image: string;
 }
 
 interface EventArtists {
-    id: number,
-    events_id: number,
-    artists_id: ArtistProps,
+  id: number;
+  events_id: number;
+  artists_id: ArtistProps;
 }
 
 interface EventProps {
-    id: number,
-    start_time: string,
-    end_time: string,
-    location: string,
-    day: number,
-    artists: EventArtists[]
+  id: number;
+  start_time: string;
+  end_time: string;
+  location: string;
+  day: number;
+  artists: EventArtists[];
 }
 
 export interface DayProps {
-    id: number,
-    day: number,
-    events: EventProps[],
-    date: Date | string,
+  id: number;
+  day: number;
+  events: EventProps[];
+  date: Date | string;
+}
+
+export interface NewsProps {
+  id: number;
+  name: string;
+  date_created: Date;
```

Comment:
API zwraca tu stringa, więc trzeba na to uwazac, w miare bezpiecznie moze byc jakos tak
```suggestion
  date_created: Date | string;
```
albo bezpieczniej
```suggestion
  date_created: string;
  dateTime_created: Date;
```